From 89f82174276d41ad29f694a24be58d91bfa85280 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Mon, 30 Jul 2018 20:25:22 +0200
Subject: [PATCH 10/13] LineageParts: Remove PrivacySettings

* These only contain protected apps
* Protected apps are not coming back as of now
* This also removes protected app from settings search,
  which crashes due to the missing activity

BUGBASH-1621

Change-Id: Id2f49c1389c8eda9e932547a63bb3765db867ec5
---
 AndroidManifest.xml                           | 26 --------------
 res/xml/parts_catalog.xml                     |  5 ---
 res/xml/privacy_settings.xml                  | 32 -----------------
 .../lineageparts/PrivacySettings.java         | 35 -------------------
 4 files changed, 98 deletions(-)
 delete mode 100644 res/xml/privacy_settings.xml
 delete mode 100644 src/org/lineageos/lineageparts/PrivacySettings.java

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index a039258..fa361fd 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -119,32 +119,6 @@
                 android:resource="@string/live_display_summary" />
         </activity-alias>
 
-        <!-- Privacy settings (dashboard) -->
-        <!--
-        <activity-alias
-            android:name=".PrivacySettings"
-            android:label="@string/privacy_settings_title"
-            android:targetActivity="PartsActivity">
-            <intent-filter android:priority="3">
-                <action android:name="com.android.settings.action.EXTRA_SETTINGS" />
-                <action android:name="org.lineageos.lineageparts.PRIVACY_SETTINGS" />
-                <category android:name="android.intent.category.DEFAULT" />
-            </intent-filter>
-            <meta-data
-                android:name="com.android.settings.category"
-                android:value="com.android.settings.category.personal" />
-            <meta-data
-                android:name="com.android.settings.icon"
-                android:resource="@drawable/ic_settings_privacy" />
-            <meta-data
-                android:name="org.lineageos.settings.summary.receiver"
-                android:value="org.lineageos.lineageparts.PartsUpdater" />
-            <meta-data
-                android:name="org.lineageos.settings.summary.key"
-                android:value="privacy_settings" />
-        </activity-alias>
-        -->
-
         <!-- Button settings (System category) -->
         <activity-alias
             android:name=".input.ButtonSettings"
diff --git a/res/xml/parts_catalog.xml b/res/xml/parts_catalog.xml
index da07250..2ad4fdb 100644
--- a/res/xml/parts_catalog.xml
+++ b/res/xml/parts_catalog.xml
@@ -54,11 +54,6 @@
           android:fragment="org.lineageos.lineageparts.notificationlight.NotificationLightSettings"
           lineage:xmlRes="@xml/notification_light_settings" />
 
-    <part android:key="privacy_settings"
-          android:title="@string/privacy_settings_title"
-          android:fragment="org.lineageos.lineageparts.PrivacySettings"
-          lineage:xmlRes="@xml/privacy_settings" />
-
     <part android:key="profiles_settings"
           android:title="@string/profiles_settings_title"
           android:fragment="org.lineageos.lineageparts.profiles.ProfilesSettings"
diff --git a/res/xml/privacy_settings.xml b/res/xml/privacy_settings.xml
deleted file mode 100644
index 7ad0a3f..0000000
--- a/res/xml/privacy_settings.xml
+++ /dev/null
@@ -1,32 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-     Copyright (C) 2016 The CyanogenMod Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
-        android:title="@string/privacy_settings_title"
-        android:key="privacy_settings">
-
-    <!-- Protected apps -->
-    <Preference
-        android:title="@string/protected_apps_manager_title"
-        android:summary="@string/protected_apps_manager_summary">
-        <intent
-            android:action="android.intent.action.VIEW"
-            android:targetPackage="com.android.settings"
-            android:targetClass="com.android.settings.applications.ProtectedAppsActivity" />
-    </Preference>
-
-</PreferenceScreen>
diff --git a/src/org/lineageos/lineageparts/PrivacySettings.java b/src/org/lineageos/lineageparts/PrivacySettings.java
deleted file mode 100644
index d654634..0000000
--- a/src/org/lineageos/lineageparts/PrivacySettings.java
+++ /dev/null
@@ -1,35 +0,0 @@
-/*
- * Copyright (C) 2016 The CyanogenMod Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.lineageos.lineageparts;
-
-import android.content.Context;
-import android.os.Bundle;
-
-import lineageos.providers.LineageSettings;
-
-/**
- * Privacy settings
- */
-public class PrivacySettings extends SettingsPreferenceFragment {
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-
-        addPreferencesFromResource(R.xml.privacy_settings);
-    }
-}
-- 
2.17.1

